### PR TITLE
feature(nemesis): new manager nemesis that backs up specific kespaces

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
https://trello.com/c/vs8ZCLbK/2319-backup-snapshot-multiple-tables-nemesis-using-manager

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -1459,8 +1459,15 @@ def disrupt_modify_table(self):
         disrupt_func = getattr(self, disrupt_func_name)
         disrupt_func()
 
+    def disrupt_mgmt_backup_specific_keyspaces(self):
+        self._set_current_disruption('ManagementBackupWithKFlag')
```
Comment (fruch) [inline sdcm/nemesis.py:1463]: s/ManagementBackupWithKFlag/ManagementBackupWithSpecificKeyspaces/

don't force people to mentally remember what K flag is

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -1459,8 +1459,15 @@ def disrupt_modify_table(self):
...
+        self._mgmt_backup(use_k_flag=True)
+
     def disrupt_mgmt_backup(self):
         self._set_current_disruption('ManagementBackup')
+        self._mgmt_backup(use_k_flag=False)
```
Comment (fruch) [inline sdcm/nemesis.py:1468]: again k has a specific meaning, writing fulling...

File: sdcm/nemesis.py (hunk 1 above)
Comment (fruch) [reply sdcm/nemesis.py:1463]: should match the function name, for clarify

**Review by fruch — CHANGES_REQUESTED** [review]
mentioning K in function name or arguments is too cryptic
please use full name, so it would be clear

**Comment by roydahan** [comment]
I don't see any link to staging jobs that shows it works on longevities multiple times.

**Comment by ShlomiBalalis** [comment]
@roydahan https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/Shlomo/job/test-3h-backup-k/

**Review by fruch — APPROVED** [review]
LGTM

**Comment by roydahan** [comment]
@ShlomiBalalis / @fruch once it pass in a short / long longevity in master, let's backport to 4.3.
We need to see it tested on releases that are out.
(maybe even 4.2).

**Pull request merged**